terser-webpack-plugin used in webpack.conf.js but not declared as a dependency

### Description

webpack.conf.js requires terser-webpack-plugin at the top level:


const TerserPlugin = require('terser-webpack-plugin');
However, terser-webpack-plugin is not listed anywhere in package.json. It was previously available implicitly because webpack 5 used to bundle it as one of its own dependencies. As of webpack 5.108.0, webpack no longer includes it, causing the build to fail with:


Error: Cannot find module 'terser-webpack-plugin'
Require stack:
- /path/to/node_modules/prebid.js/webpack.conf.js
- /path/to/node_modules/prebid.js/gulpfile.js
Steps to reproduce

Install prebid.js 11.8.0 fresh (no lockfile)
Run gulp serve or gulp build
yarn/npm resolves webpack to 5.108.0 (the latest compatible with ^5.103.0)
Build fails with Cannot find module 'terser-webpack-plugin'
Expected behavior

terser-webpack-plugin should be explicitly declared in devDependencies so the build works regardless of which patch version of webpack is resolved.

### Suggested fix

Add terser-webpack-plugin to devDependencies in package.json:


"terser-webpack-plugin": "^5.3.0"
Environment

prebid.js: 11.8.0
webpack resolved version: 5.108.0
Node: >=20

cc @Jacobkmiller

--- a/package.json
+++ b/package.json
@@ -117,6 +117,7 @@
     "puppeteer": "^24.10.0",
     "resolve-from": "^5.0.0",
     "sinon": "^22.0.0",
+    "terser-webpack-plugin": "^5.6.1",
     "typescript": "^5.8.2",
     "typescript-eslint": "^8.26.1",
     "url": "^0.11.0",

